Controller/Controller.php

```diff
@@ -164,10 +165,16 @@ public function getTranslationsAction(Request $request, $domain, $_format)
             }
         }
 
+        if (version_compare(Kernel::VERSION, '2.7.0', '>=')) {
```

Comment:
This is a very bad idea. Someone using components directly can have a different version for the Config component and the Kernel. You need to detect whether the Config component supports the new API, not what the kernel version is